qutip/core/operators.py

```diff
@@ -18,6 +18,7 @@
 
 from . import data as _data
 from .qobj import Qobj
+from .tensor import tensor
```

Comment:
You could move the importation inside the function to avoid circular importation.

Reply:
done, thanks!